capten/cassandra/migrations/006_cross_plane.up.cql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE CrossplaneProviders (
+    id UUID,
+    cloud_type TEXT,
+    provider_name TEXT,
+    cloud_provider_id TEXT,
```

Comment:
for workflow handling

add below filed

workflow_id TEXT
workflow_status TEXT